app/models/application_form.rb

```diff
@@ -100,6 +100,7 @@ def equality_and_diversity_answers_provided?
     international_address country postcode equality_and_diversity
     work_history_breaks first_nationality second_nationality third_nationality
     fourth_nationality fifth_nationality phone_number
+    recruitment_cycle_year
```

Comment:
These tests are made trickier to maintain after this change because the serialiser depends on a different set of attributes depending on whether you are using the old or new model.